polytope/polytope.py

```diff
@@ -964,12 +964,18 @@ def is_convex(reg, abs_tol=ABS_TOL):
 
 
 def is_inside(polyreg, point, abs_tol=ABS_TOL):
-    """Return `point in polyreg`."""
+    """Return `point in polyreg`.
+
+    @type point: `collections.abc.Sequence` or `numpy.ndarray`
```

Comment:
As a comment to be addressed after this pull request is merged, I want to review the documentation and add non-formal explanations that make usage instructions more accessible to some readers.

For example, referring to an abstract base class `Sequence` is precise and therefore important for a function docstring. However, if the user is new to Python, then they might wonder whether this is a special requirement beyond what they intuitively expect to be a "sequence".

Reply:
I agree that writing `collections.abc.Sequence` could raise questions, an alternative
would be to write `list` and implicitly suggest that duck typing would work,
or write "`list`, or `numpy.ndarray`, or equivalent".